django/src/rdwatch/utils/stac_search.py

```diff
@@ -53,35 +56,23 @@ def _fmt_time(time: datetime):
     return f'{time.isoformat()[:19]}Z'
 
 
-def landsat_search(
-    timestamp: datetime,
-    bbox: tuple[float, float, float, float],
-    timebuffer: timedelta | None = None,
-    page: int = 1,
-) -> Results:
-    url = 'https://landsatlook.usgs.gov/stac-server/search'
-    if timebuffer is not None:
-        min_time = timestamp - timebuffer
-        max_time = timestamp + timebuffer
-        time_str = f'?datetime={_fmt_time(min_time)}/{_fmt_time(max_time)}'
-    else:
-        time_str = f'?datetime={_fmt_time(timestamp)}'
-    url += time_str
-    url += f"&bbox={','.join(str(coord) for coord in bbox)}"
-    url += '&collections=landsat-c2l1,landsat-c2l2-sr'
-    url += f'&page={page}'
-    url += '&limit=100'
-    with urlopen(url) as resp:
-        return json.loads(resp.read())
+COLLECTIONS: dict[str, list[str]] = {
+    'L8': ['landsat-c2l1', 'landsat-c2l2-sr'],
+    'S2': [],
+}
+
+if settings.ACCENTURE_IMAGERY_VERSION is not None:
+    COLLECTIONS['S2'].append(f'ta1-s2-acc-{settings.ACCENTURE_IMAGERY_VERSION}')
 
 
-def sentinel_search(
+def stac_search(
+    source: Literal['S2', 'L8'],
     timestamp: datetime,
     bbox: tuple[float, float, float, float],
     timebuffer: timedelta | None = None,
     page: int = 1,
 ) -> Results:
-    url = 'https://earth-search.aws.element84.com/v0/search'
```

Comment:
Can we leave this URL as a comment, I remember Matt saying after the program we may want to revert back to public servers for serving images.